jdaviz/core/template_mixin.py

```diff
@@ -3154,6 +3174,15 @@ def _on_glue_value_changed(self, value):
             # to be an Any traitlet in order to handle "Custom"
             value = type(self.value)(value)
         self.value = value
+
+        if self._glue_name == 'stretch_parameters':
+            # stretch_parameters is a dictionary so won't trigger the @observe on
+            # _update_stretch_curve, so we'll need to manually call it (and make sure
+            # it is not skipped if it has already been called since an actual traitlet change)
+            if '_update_stretch_curve' in self.plugin._methods_skip_since_last_active:
+                self.plugin._methods_skip_since_last_active.remove('_update_stretch_curve')
```

Comment:
I don't _think_ this is needed anymore with the changes at line 240 that avoids skipping if no message is passed.

```suggestion
```